### PR TITLE
Feature: Setup user group page

### LINKED ISSUE #10: Setup user groups page
## Description

blocked by #6 

Add in the drawer an item : `User groups`, on click that go the UserGroupsPage (`/user-groups`).

On user groups page only display `User groups` title.

image

### PR DESCRIPTION
**Pull request by DimitriChauvel**
Resolve #10

### CODE REVIEW

File: src/components/drawer/ApplicationDrawer.vue [hunk 1]
```diff
@@ -43,6 +43,11 @@ const menuList = ref([
     label: t('ApplicationDrawer.text.users'),
     url: '/users',
   },
+  {
+    icon: t('ApplicationDrawer.icon.user-groups'),
```
Comment (k-y0u) [inline src/components/drawer/ApplicationDrawer.vue:47]: userGroups

File: src/components/drawer/ApplicationDrawer.vue [hunk 2]
```diff
@@ -43,6 +43,11 @@ const menuList = ref([
...
+    label: t('ApplicationDrawer.text.user-groups'),
```
Comment (k-y0u) [inline src/components/drawer/ApplicationDrawer.vue:48]: userGroups

File: src/i18n/en-US/index.js [hunk 3]
```diff
@@ -2,11 +2,13 @@ export default {
   ApplicationDrawer: {
     text: {
       users: 'Users',
+      'user-groups': 'User Groups',
```
Comment (k-y0u) [inline src/i18n/en-US/index.js:5]: userGroups

File: src/i18n/en-US/index.js [hunk 4]
```diff
@@ -2,11 +2,13 @@ export default {
...
       libraries: 'Libraries',
       roles: 'Roles',
     },
     icon: {
       users: 'fa-solid fa-user',
+      'user-groups': 'fa-solid fa-users',
```
Comment (k-y0u) [inline src/i18n/en-US/index.js:11]: userGroups

File: src/pages/UserGroupsPage.vue [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<template>
+  <q-page class="flex q-pl-md bg-grey-1">
+    <h4
+      class="q-ma-none q-mt-md"
+      data-cy="page_user-groups_title"
```
Comment (k-y0u) [inline src/pages/UserGroupsPage.vue:5]: userGroups

File: tests/e2e/features/Application.feature [hunk 6]
```diff
@@ -7,6 +7,11 @@ Feature: Test roundtrip of the application
     And  I expect '[data-cy="page_users_title"]' is 'Users'
     And  I expect current url is '/users'
 
+    # Navigate to UserGroupsPage
+    When I click on '[data-cy="drawer_item_User\ Groups"]'
+    Then I expect '[data-cy="page_user-groups_title"]' is 'User Groups'
```
Comment (k-y0u) [inline tests/e2e/features/Application.feature:12]: userGroups

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/drawer/ApplicationDrawer.vue [hunk 7]
```diff
@@ -43,6 +43,11 @@ const menuList = ref([
     label: t('ApplicationDrawer.text.users'),
     url: '/users',
   },
+  {
```
Comment (k-y0u) [inline src/components/drawer/ApplicationDrawer.vue:46]: add a variable `name` to each item of the list to use it in `data-cy` instead of `label`

**Review by k-y0u — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**